### ISSUE TITLE
[Accessibility] Narrator cannot announce Expand/Collapse state for properties with multiline textbox typeeditor drop-down 

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#6656]
**.NET Core Version:** 

* Main: 7.0.100-preview.2.22108.9
* Release: 7.0.100-preview.1.22108.11

**Have you experienced this same bug with .NET Framework?** 

Yes

**Problem description:**

Narrator doesn't announce the expanded state when pressing F4 or 'Alt + down' for such properties which drop-down control is RichEdit control after expanded them. In this case, now Narrator focus in the drop-down list: RichEdit instead of this entire property: 'Text' as following screenshot:
![bug](https://user-images.githubusercontent.com/94418985/153158605-ffc7feb9-e1d8-40c0-a3e5-6724fad37a63.png)

And check the related properties in Inspect, the ExpandCollapse.ExpandCollapseState exists and displays Expand state --- It's correct.
![in](https://user-images.githubusercontent.com/94418985/153163906-25687138-3abc-419f-99f5-819a181bd0bb.png)

**Expected behavior:**

Narrator should announce the expanded state.

**Minimal repro:**

1. Create a winforms core app with PropertyGrid and Button control
2. Associate the Button with PropertyGrid as a selected object
3. Build and run the core project
4. Start Narrator tool
5. Press F4 to open the RichEdit control of Text property

**More info:**

1. This issue cannot repro from 7.0.100-alpha.1.21566.20 to 7.0.100-alpha.1.21568.2.
2. This has been fixed and added [comment](https://github.com/dotnet/winforms/issues/3672#issuecomment-953431994) in [#3672](https://github.com/dotnet/winforms/issues/3672).

**Comment by Tanya-Solyanik** [comment]
@MelonWang1  - does the same issue reproduce in JAWS and NVDA?

**Comment by MelonWang1** [comment]
@Tanya-Solyanik For JAWS, it is locked out, hitting known issue [6704](https://github.com/dotnet/winforms/issues/6704).
For NVDA, it reproduces the issue, see screenshot:
![nvda1](https://user-images.githubusercontent.com/94418985/155251787-6d7ac502-4126-4099-8391-8b2c0d35d74b.png)

**Comment by MandiMan** [comment]
Verified on .NET 7.0 test pass build: .NET 7.0.100-rc.1.22425.12, issue was fixed as below:
![issue6657](https://user-images.githubusercontent.com/108860782/187140456-bcedb5d4-7cf0-4d0b-b060-58981b184f54.png)

**Comment by MandiMan** [comment]
Verified on .NET 8.0 latest build from main branch: NET 8.0.100-alpha.1.22451.15, issue was fixed, test result is same as above screenshot.
